integration-tests/generate_from_templates.py

```diff
@@ -0,0 +1,105 @@
+parser.add_argument('-u', dest='auth_type', type=str, default='None', choices=['Basic', 'None'], help='auth type for bob')
+parser.add_argument('--index-memory-limit', dest='index_memory_limit', type=str, default='8 GiB', help='memory limit for all indexes')
+parser.add_argument('--enable-aio', dest='enable_aio', type=str, default='true', choices=['true', 'false'], help='enables linux AIO.')
+parser.add_argument('-p', dest='disks_events_logfile', type=str, default='/bob/log/bob_events.csv', help='path to logfile with info about disks states switches.')
+parser.add_argument('-b', dest='max_blob_size', type=str, default='100mb')
+parser.add_argument('--allow-duplicates', dest='allow_duplicates', type=str, default='true', choices=['true', 'false'], help='disables search for existing keys before write.')
+parser.add_argument('-d', dest='max_data_in_blob', type=int, default=10000)
+parser.add_argument('--blob-file-name-prefix', dest='blob_file_name_prefix', type=str, default='bob')
+parser.add_argument('--fail-retry-timeout', dest='fail_retry_timeout', type=str, default='100ms', help='retry to reinit pearl backend after fail.')
+parser.add_argument('--alien-disk', dest='alien_disk', type=str, default='d1')
+parser.add_argument('--bloom-filter-max-buf-bits-count', dest='bloom_filter_max_buf_bits_count', type=int, default=10000, 
+help='sets bloom filter buffer size in bits count, best value ~= max_data_in_blob.')
+parser.add_argument('--root-dir-name', dest='root_dir_name', type=str, default='bob')
+parser.add_argument('--alien-root-dir-name', dest='alien_root_dir_name', type=str, default='/bob/data/d1/alien')
+parser.add_argument('-t', dest='timestamp_period', type=str, default='1m', help='period when new pearl directory created.')
+parser.add_argument('--create-pearl-wait-delay', dest='create_pearl_wait_delay', type=str, default='100ms', help='each thread will wait this period if another thread creating pearl.')
+parser.add_argument('--metrics-name', dest='metrics_name', type=str, default='bob', help='add base name for metrics.')
+parser.add_argument('--graphite-enabled', dest='graphite_enabled', type=str, default='false', choices=['true', 'false'])
+parser.add_argument('--prometheus-enabled', dest='prometheus_enabled', type=str, default='false', choices=['true', 'false'])
+parser.add_argument('--path', dest='path', type=str, help='sets path to directory where configs will be generated.', default='/tmp')
+parser.add_argument('-min_port', dest='min_port', type=int, required=True, help='Port of the first bob container.')
+
+args = parser.parse_args()
+
+path = os.path.join(pathified(args.path),'generated_configs')
+
+#get the offset for bobp operations and set in as env 
+bobp_actions_offset = float(str(Quantity(args.check_interval, scale='s')).replace(' s', '')) + 1
+with open(os.getenv('GITHUB_ENV'), 'a') as file:
+    file.write(f"BOB_BOBP_ACTIONS_OFFSET={bobp_actions_offset}")
```

Comment:
This is not obvious. As I wrote before, it is better to remove this and pass hardcoded values as parameters